proto/substrait/plan.proto

```diff
@@ -24,6 +24,10 @@ message PlanRel {
 // Describe a set of operations to complete.
 // For compactness sake, identifiers are normalized at the plan level.
 message Plan {
+  // Substrait version of the plan. Optional up to 0.16.0, required for later
```

Comment:
If this isn't merged this week, I'll probably have to update this to track the version that would introduce this.